src/Microsoft.HttpRepl/OpenApi/OpenApiDotNetApiDefinitionReader.cs

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.OpenApi.Models;
+using Microsoft.OpenApi.Readers;
+
+namespace Microsoft.HttpRepl.OpenApi
+{
+    internal class OpenApiDotNetApiDefinitionReader : IApiDefinitionReader
+    {
+        public bool CanHandle(string document)
+        {
+            OpenApiStringReader reader = new();
```

Comment:
Yep. Amusingly I wrote most of this code (or one of them, not sure which) with `var` and then had VS convert it to explicit... and then remembered C# 9 had target-typed new. So that one line of code went through a lot of change for being so simple.